charts/chatbot/values.yaml

```diff
@@ -6,8 +6,7 @@ replicaCount: 1
 
 image:
   repository: jenkinsciinfra/ircbot
-  tag: 165-buildbc1736
-  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
```suggestion
```
To be coherent with https://github.com/jenkins-infra/public-charts/blob/main/charts/accountapp/values.yaml#L7-L8